### PR TITLE
Fix arrows and dashes

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Needs #591 to be merged first. I'll rebase on top once that's done.

### CODE REVIEW

**Review by plural — APPROVED** [review]
looks good.  maybe we should add a check in the validation script for the wrong forms in text and stripped_text in a followup?

**Comment by NoahTheDuke** [comment]
Sure, I can add that!

File: validate.py [hunk 1]
```diff
@@ -192,6 +192,54 @@ def validate_card(args, card, card_schema, pack_code, factions_data, types_data,
...
+            "Stripped text contains non-ascii characters in card: (pack code: '{}' title: '{}' stripped_text '{}')\n".format(
+                pack_code,
+                card['title'],
+                stripped_text,
+            ),
+            0,
+        )
+        validation_errors += 1
+
+def verify_text_has_fancy_text(args, card, pack_code):
```
Comment (plural) [inline validate.py:214]: teehee

File: pack/draft.json [hunk 2]
```diff
@@ -169,7 +169,7 @@
         "position": 13,
         "quantity": 1,
         "side_code": "corp",
-        "stripped_text": "Draft format only. If you have more cards rezzed than any other faction, when the Runner's turn begins, place an advancement token on a piece of ice.",
+        "stripped_text": "Draft format only. If you have more  cards rezzed than any other faction, when the Runner's turn begins, place an advancement token on a piece of ice.",
```
Comment (NoahTheDuke) [inline pack/draft.json:172]: oops, one sec

File: validate.py [hunk 3]
```diff
@@ -192,6 +192,54 @@ def validate_card(args, card, card_schema, pack_code, factions_data, types_data,
...
+            args,
+            "Incorrect interrupt text in card: (pack code: '{}' title: '{}' text '{}')\n".format(
+                pack_code,
+                card['title'],
+                text,
+            ),
+            0,
+        )
+        validation_errors += 1
+    if ('Interface' in text) and (('Interface ->' in text) or ('Interface →' not in text)):
```
Comment (plural) [inline validate.py:230]: why accept both formats instead of just the one?

**Review by plural — COMMENTED** [review]
just one question

File: validate.py (hunk 3 above)
Comment (NoahTheDuke) [reply validate.py:230]: This is checking the "text" field, so it should be "Interface →" (with the unicode arrow). I'm raising an error if it's a dash and greater than or anything that's not a unicode error.

**Review by plural — APPROVED** [review]

**Pull request merged**